ProcessMessage/handler.ts

```diff
@@ -384,6 +452,26 @@ export const getProcessMessageHandler = ({
 
               return result;
             }),
+            TE.chain(blockedInboxOrChannel => {
+              if (
+                createdMessageEvent.senderMetadata.serviceCategory ===
+                SpecialServiceCategoryEnum.SPECIAL
+              ) {
+                // If the service has category equals to SPECIAL, the INBOX value in blockedInboxOrChannel depends from the Activation
+                // value. If the SPECIAL service is ACTIVE (exists an activation for couple user/service with ACTIVE status)
+                // we remove INBOX from blockedInboxOrChannel, otherwise we add the INBOX value.
+                return getActivationForSpecialServices(lActivation)({
+                  blockedInboxOrChannel,
+                  context,
+                  createdMessageEvent,
```

Comment:
it looks like you only need the `senderServiceId`

Reply:
Fixed in 0b1169e